### DISCUSSION TITLE
Hotkey for linking elements/units

### DISCUSSION THREAD

**Discussion by sooooosh** [diesel-modding/BeardLib-Editor#611] [Implemented Ideas]
**Is your feature request related to a problem? Please describe.**
Right now linking elements and units requires selecting the element/unit and opening a menu with a list of all the units or elements, either filtering the list or looking for the correct one to add. This takes up a lot of time and gets annoying when you need to link a lot of stuff. Depending on the amount of elements/units already in the level it can also take some time to load the list every time.

**Describe the solution you'd like**
As Ilija described in his recent streams, BoW has a feature that lets you link up elements with a hotkey. The same, or a similar feature would speed up map development for BLE a lot. Select an element, hold down the hotkey and while it's being pressed click on the element you want to add to the "On Executed List".
The hotkey would of course be customizable, by default I would probably put it to "x" for easy access. 

The same could also be made for the elements with an Event List, Sequence List or Element List like CounterTrigger or UnitSequence elements. Maybe "M" for the hotkey in this case.

If the list has multiple choices, like events for Instances or Sequences for units, it would just add the element without anything specified in there.

**Comment by Luffyyy** [comment]
https://user-images.githubusercontent.com/11510099/120064102-57199b00-c073-11eb-94b9-78e7c23a0fa3.mp4

With the little time I had, I (partly) implemented this. Basically, only on executed and cannot be customized. https://wiki.modworkshop.net/books/payday-2/page/hot-keys
